src/components/projekti/suunnitteluvaihe/komponentit/VuorovaikutustilaisuusDialog/TilaisuudenNimiJaAika.tsx

```diff
@@ -17,6 +18,8 @@ export default function TilaisuudenNimiJaAika(props: { index: number; mostlyDisa
     trigger,
   } = useFormContext<VuorovaikutustilaisuusFormValues>();
 
+  useAikavalidointi(props.index);
+
```

Comment:
Tää hookki triggeröi nyt päättymispäivää muuttaessa toistamiseen sen validoinnin. Suosittelisin tekemään sen triggeröinnin. Alkamisaikakentälle näin:

```
        <TimePicker
          disabled={props.mostlyDisabled}
          label="Alkaa *"
          {...register(`vuorovaikutusTilaisuudet.${props.index}.alkamisAika`, {
            onChange: () => {
              trigger(`vuorovaikutusTilaisuudet.${props.index}.paattymisAika`);
            },
          })}
          error={(errors as any)?.vuorovaikutusTilaisuudet?.[props.index]?.alkamisAika}
        ></TimePicker>

```
Eli ainoastaan tarvii lisätä toi registerOptions tolle register-funktiolle, ja sille onChange-funktion, joka triggeröi sen validoinnin.